src/pages/LoginPage.js

```diff
@@ -8,36 +8,62 @@ import LoginForm from '../components/users/LoginForm';
 import AppStoreContainer from '../components/app-store-container/AppStoreContainer';
 import menuIcon from '../assets/icons/menuIcon.png';
 import { homePageLink } from '../constants/link.constants';
+import SignUpOptions from '../components/users/SignUpOptions';
+import ForgotPwd from '../components/users/ForgotPwd';
 
 const LoginPage = () => {
   const intl = useIntl();
 
+  const [forgotPwd, setForgotPwd] = useState(false);
+
+  const handleForgotPwd = () => {
+    setForgotPwd(true);
+  };
+
   const user = localStorage.getItem('user');
   if (user) {
     return <Redirect to={homePageLink} />;
   }
 
   return (
     <div className="main-div">
-      <div className="left-container">
-        <div className="left-container__menu">
+      <div className="login-page-container">
+        <div className="login-page-container__menu">
           <img src={menuIcon} alt="menu_button" />
         </div>
-        <div className="left-container__smilies">
+        <div className="login-page-container__smilies">
           <img src={smilies} alt="smilies" />
         </div>
-        <p className="left-container__app-title">TARGET MVD</p>
-        <p className="left-container__app-subtitle">
+        <p className="login-page-container__app-title">TARGET MVD</p>
+        <p className="login-page-container__app-subtitle">
           {intl.formatMessage({
             id: 'loginpage.subtitle.text',
           })}
         </p>
-        <p className="left-container__description-text">
-          {intl.formatMessage({
-            id: 'loginpage.description.text',
-          })}
-        </p>
-        <LoginForm />
+        {!forgotPwd ? (
+          <div align="center">
+            <p className="login-page-container__description-text">
+              {intl.formatMessage({
+                id: 'loginpage.description.text',
+              })}
+            </p>
+            <LoginForm intl={intl} />
+            <div className="user-form__forgot-pwd">
+              <button
+                type="button"
+                className="forgot_pwd__btn-text"
+                onClick={handleForgotPwd}
```

Comment:
Since handleForgotPwd only call setForgotPwd, there is no need to define it as a constant. 
```suggestion
                onClick={() => setForgotPwd(true)}
```